src/index/txindex.cpp

```diff
@@ -10,6 +11,8 @@
 #include <validation.h>
 #include <warnings.h>
 
+constexpr int64_t SYNC_LOG_INTERVAL = 30; // seconds
```

Comment:
Correct me if I'm wrong, but I assume this could be inlined or something by the compiler if it were not declared static. I tried looking it up briefly and it said something along those lines, but then I got bored because I don't care all that much (something, something ODR-rule?). Will change if you still want.

Reply:
https://github.com/bitcoin/bitcoin/pull/11857#discussion_r174594791

> Correct me if I'm wrong, but I assume this could be inlined or something by the compiler if it were not declared static. I tried looking it up briefly and it said something along those lines, but then I got bored because I don't care all that much (something, something ODR-rule?). Will change if you still want.

Hmm, I guess you @eklitzke are making me question my sanity, but why would adding `static` here prevent inlining, or "reserve .bss space" or do anything along these lines?

If this isn't declared static, the compiler doesn't know that there isn't some code in another object file that will access the SYNC_LOG_INTERVAL variable or take its address. So there will necessarily be a SYNC_LOG_INTERVAL symbol in the object file. If it's static, it can just be internal to the object.